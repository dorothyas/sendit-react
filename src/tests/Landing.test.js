import React from 'react';
import renderer from 'react-test-renderer';
import Landing from '../components/Landing';

describe('Landing page', () => {
    it('should render without crashing', () => {
        const wrapper = renderer.create(<Landing />);
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
