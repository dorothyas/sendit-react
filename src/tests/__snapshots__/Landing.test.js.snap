// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page should render without crashing 1`] = `
<h1>
  Landing Page
</h1>
`;
